shopfloor_mobile_base/static/wms/src/router.js

```diff
@@ -32,9 +26,16 @@ const routes = [
     // TODO Fix this it needs to be the last route, but I think it is not anymore with the dynamic one added.
     // { path: '*', component: NotFound },
 ];
+const default_home_route = {
+    path: "/",
+    component: HomePage,
+    name: "home",
+    meta: {requiresAuth: true, requiresProfile: true},
```

Comment:
@JuMiSanAr you must set `isHomeRoute` here otherwise your filter on login won't work